api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1043,29 +1078,30 @@ ml_check_nnfw_availability (ml_nnfw_type_e nnfw, ml_nnfw_hw_e hw,
 
   switch (nnfw) {
     case ML_NNFW_TYPE_CUSTOM_FILTER:
+      fw = nnstreamer_filter_find ("custom");
       /* Always available */
```

Comment:
if it's always available, how about adding `assert`? (because line 1114 implies a potential segmentation fault)

Reply:
done.